Packs/MicrosoftGraphAPI/.pack-ignore

```diff
@@ -1,2 +1,2 @@
 [file:MicrosoftGraphAPI.yml]
```

Comment:
```suggestion
```

Reply:
Because of changes made before, which replaced parameters with different names, it is not possible to delete the ignoring of error IN124 here.